src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/Decision.java

```diff
@@ -54,6 +54,11 @@ public void setDecider(String decider) {
     this.decider = decider;
   }
 
+  @Override
+  public String toString() {
+    return decider + actions;
```

Comment:
nit: `decider + ": " + actions` ? (assuming this is for logging?)

Reply:
done